changelogs/fragments/2083-backup_plan_info-bugfix-get-info-for-all-plans.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
I don't know if this is a bugfix. This feature was not working because it was not implemented and the module does not say it supports listing all backup plans (that's why  `backup_plan_names` was a required parameter). To me it is probably minor_changes, a new functionality. You should also document that when O(backup_plan_names) is omitted the module gathers information about all backup plans.